config/formats.ts

```diff
@@ -153,11 +158,16 @@ export const Formats: FormatList = [
 		banlist: ['Koraidon', 'Miraidon', 'Commander', 'Focus Sash', 'King\'s Rock', 'Ally Switch', 'Final Gambit', 'Perish Song', 'Swagger'],
 	},
 	{
-		name: "[Gen 9] Battle Stadium Doubles",
+		name: "[Gen 9] VGC 2023 Series 1",
 
 		mod: 'gen9',
 		gameType: 'doubles',
-		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Min Source Gen = 9', 'VGC Timer'],
+		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Paldea Pokedex', 'Min Source Gen = 9', 'VGC Timer'],
+		banlist: [
+			'Great Tusk', 'Brute Bonnet', 'Sandy Shocks', 'Scream Tail', 'Flutter Mane', 'Slither Wing', 'Roaring Moon',
+			'Iron Treads', 'Iron Moth', 'Iron Hands', 'Iron Jugulis', 'Iron Thorns', 'Iron Bundle', 'Iron Valiant',
+			'Ting-Lu', 'Chienpao', 'Wo-Chien', 'Chi-Yu',
+		],
```

Comment:
Wouldn't the proper route be to give them the Sub-Legendary tag in `pokedex.ts`?

Reply:
Are the Paradox mons classified Sub-Legendary? If not, it doesn't really change a whole lot.